src/mlstacks/analytics/client.py

```diff
@@ -34,10 +34,24 @@
 logger = getLogger(__name__)
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
logging.getLogger("segment").disabled = True
```

Reply:
Great! I wasn't sure if disabling the segment logger would inadvertently suppress all error logs, and potentially prevent the `on_error` trigger. However, it's reassuring to know it still functions as intended based on your tests. Thanks for thoroughly examining this.  I'll commit the changes.